scripts/github-status/github-status.rb

```diff
@@ -152,17 +157,11 @@ def show_forcerebuild_pull_requests
     raise if options[:pr].nil? || options[:pr].empty?
     puts ghc.get_pr_latest_sha(options[:pr])
   when 'set-status'
-    raise unless ['success', 'failure', 'error', 'pending'].include? options[:status]
+    raise unless [:success, :failure, :error, :pending].include? options[:status]
     raise if options[:sha].nil? || options[:sha].empty?
-    raise if options[:target_url].nil? || options[:target_url].empty?
-    case options[:status]
-      when 'success'
-        ghc.create_success_status(options[:sha], options[:target_url])
-      when 'failure', 'error'
-        ghc.create_failure_status(options[:sha], options[:target_url])
-      when 'pending'
-        ghc.create_pending_status(options[:sha], options[:target_url])
-    end
+    msg = options[:message]
```

Comment:
In that case you can skip this variable definition and put ```options[:message]``` and ```options[:target_url]``` directly as a method parameter.